### ISSUE TITLE
Error in prediction/cv

### ISSUE THREAD

**Issue by salimamoukou** [bmihaljevic/bnclassify#59]
Hi, 

I always have this error when i try to predict/cv:  "in compile_grain(params(x)) : gRain currently incompatible".

**Comment by bmihaljevic** [comment]
Hi,

could you tell me which version of bnclassify you are using? This message not appear solved in the latest version.

**Comment by salimamoukou** [comment]
I use the 0.4.5

**Comment by bmihaljevic** [comment]
Indeed, I have solved this on the last version that is on github but have not uploaded it to CRAN. You can install from package from github instead: `devtools::install_github("bmihaljevic/bnclassify/")`. That should solve it.
